source/misc/bash_startup.in

```diff
@@ -1,257 +1,95 @@
-#!/bin/bash
-# This is based on "preexec.bash" but is customized for iTerm2.
+# This is inspired by "preexec.bash" but is customized for iTerm2. Sourced, so
+# no shebang. This module requires 2 bash features you must not otherwise be
+# using: the DEBUG trap and the PROMPT_COMMAND variable.
 
-# Note: this module requires 2 bash features which you must not otherwise be
-# using: the "DEBUG" trap, and the "PROMPT_COMMAND" variable.  iterm2_preexec_install
-# will override these and if you override one or the other this _will_ break.
+osc='\033]'; cso='\007' # Beginning, ending escape sequences.
```

Comment:
> Let's not use more than one command per line. I find it harder to read.

Fair enough.
>  I haven't seen \007 called "CSO" before. Where does the acronym come from?

I just wanted a short variable name and couldn't think of one. It's just OSC backwards. Was kinda thinking like `case esac`, `if fi`...